packages/providers/src/util.ts

```diff
@@ -41,11 +42,34 @@ export function sleep(time: number = 1000) {
   });
 }
 
-export const getReceiptsWithMissingOutputVariables = (
-  receipts: Array<TransactionResultReceipt>
-): Array<TransactionResultReceipt> =>
-  receipts.filter(
-    (receipt) =>
-      receipt.type === ReceiptType.Revert &&
-      receipt.val.toString('hex') === FAILED_TRANSFER_TO_ADDRESS_SIGNAL
+const getReceiptsWithMissingOutputVariables = (
+  receipt: TransactionResultReceipt
+): receipt is ReceiptRevert =>
+  receipt.type === ReceiptType.Revert &&
+  receipt.val.toString('hex') === FAILED_TRANSFER_TO_ADDRESS_SIGNAL;
+
+const getReceiptsWithMissingContractIds = (
+  receipt: TransactionResultReceipt
+): receipt is ReceiptPanic =>
+  receipt.type === ReceiptType.Panic &&
+  receipt.contractId !== '0x0000000000000000000000000000000000000000000000000000000000000000';
+
```

Comment:
Nice use of type helpers. ⚡ 

Since `getReceiptsWithMissingOutputVariables` and `getReceiptsWithMissingContractIds` don't _actually_ get a list of receipts, I suggest renaming them using question-like nomenclature.

One literal example would be:

- `doesReceiptHaveMissingOutputVariables`
- `doesReceiptHaveMissingContractIds`

Reply:
got it! this makes sense.  Will update!